lib/tizen_tpk.dart

```diff
@@ -218,27 +218,27 @@ class Signature {
   }
 }
 
-class Certificate {
-  Certificate._({
+class ProfileItem {
```

Comment:
https://docs.tizen.org/application/tizen-studio/common-tools/command-line-interface/#managing-a-security-profile

According to the Tizen CLI documentation, a _security profile_ is a _set_ of signing certificates. So I think you can use the term `Certificate` here.